apollo/src/datasources/TagResearchDataSource.js

```diff
@@ -133,7 +133,7 @@ class TagResearchDataSource extends DataSource {
     } else {
       selectedGroupIds = groupIds.groupDefault;
     }
-
+    console.log(selectedGroupIds);
```

Comment:
```suggestion
```

Reply:
Resolved in e9640ae1adabfb3a1ef966887f88cc121e22db22